examples/finegrained_plot.py

```diff
@@ -0,0 +1,76 @@
+"""
+===============================
+Fine grained Plotting Features
+===============================
+
+An example to show control over various plotting features.
+"""
+###############################################################################
+# Import the necessary modules for plotting.
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+import matplotlib.pyplot as plt
+from matplotlib.colors import LogNorm
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE
+
+
+###############################################################################
+# SkyCoord module provides flexible celestial coordinate representation and a
+# draw_limb method overplots the boundary of the spacial objects. Date of the
+# image taken can also be displayed in the plot.
+
+aiamap = sunpy.map.Map(AIA_171_IMAGE)
+
+bottom_left = SkyCoord(-400*u.arcsec, -900*u.arcsec, frame=aiamap.coordinate_frame)
+top_right = SkyCoord(800*u.arcsec, 700*u.arcsec, frame=aiamap.coordinate_frame)
+aiamap_sub = aiamap.submap(bottom_left, top_right)
+
+title_obsdate = '{:%Y-%b-%d %H:%M:%S}'.format(aiamap_sub.date)
+
+###############################################################################
+# The SunPy map peek method shows a helioprojective Cartesian grid by default.
+# This is sometimes not desired, and instead a heliographic Stonyhurst grid is
+# desired. Although maps offer a simple way to include a Stonyhurst grid, often
+# times we want more control over the Stonyhusrt grid. The example below illustrates
+# some aspects of how a map plot can be altered as desired. For more information
+# regarding the axis and grid settings, go to
+# http://docs.astropy.org/en/stable/visualization/wcsaxes/ticks_labels_grid.html
+
+fig = plt.figure(figsize=(6, 6))
+ax = plt.subplot(projection=aiamap_sub)
+aiamap_sub.plot()
+aiamap_sub.draw_limb(color='white', linewidth=2, linestyle='dashed')
+
+# To have more sensitive control over the Heliographic Stonyhurst grid, the Axes
+# object is pulled out and change its properties explicitly.
```

Comment:
I have forgotten if these comments show up in the example without a 
`###############################################################################` above them.